backend/matching-service/src/index.ts

```diff
@@ -0,0 +1,52 @@
+import cors from 'cors'
+import express, { Express, NextFunction, Request, Response } from 'express'
+import 'express-async-errors'
+import helmet from 'helmet'
+import passport from 'passport'
+import defaultErrorHandler from './middlewares/errorHandler.middleware'
+import './middlewares/passportJwt.middleware'
+import matchingRouter from './routes/matching.routes'
+
+const app: Express = express()
+
+app.use(express.urlencoded({ extended: true }))
+app.use(express.json())
+app.use(cors()) // config cors so that front-end can use
```

Comment:
if u want to use the cors package, can remove the cors middleware from lines 21 to 31